packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1873,6 +1903,20 @@ export default class NewVm extends BaseComponent {
               </Item>
             </SectionContent>
           ),
+          isAdmin && isHvm && (
+            <SectionContent>
+              <Item>
+                {hvmBootFirmware !== 'uefi' &&
+                this._templateHasBiosStrings() ? (
+                  <Tooltip content={_('templateHasBiosStrings')}>
```

Comment:
You can also add a tooltip when the boot firmware is UEFI so that the user always understands why the checkbox is disabled.